### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by nannal**
Added IPFSStore, dtube.video & hardbin

### CODE REVIEW

**Comment by koalalorenzo** [comment]
IPFSstore seems down :(

**Comment by koalalorenzo** [comment]
Now it is back up apparently 😮

File: README.md [hunk 1]
```diff
@@ -50,6 +51,7 @@ place to ask about it might be in [ipfs/apps](https://github.com/ipfs/apps) or
 * [ipfs-share](https://github.com/rameshvarun/ipfs-share) - Pastebin/Image host/File sharing application
 * [ipfs.pics](https://github.com/ipfspics/ipfspics-server) - Upload and share pics.
 * [IPFSBin](https://github.com/victorbjelkholm/ipfsbin) - Pastebin clone build.
+* [IPFSStore](https://ipfsstore.it) - File pinning for bitcoin
```
Comment (VictorBjelkholm) [inline README.md]: Seems like weird working, the pinning is not for bitcoin but rather paid with bitcoin, correct?

Comment (nannal) [inline README.md]: Yeah, you input a hash, get its associated bitcoin address, pay that and the site pins the hash for you so you know it's always online and accessible.

Comment (VictorBjelkholm) [inline README.md]: Could you change the wording to something more like: "Pinning paid with Bitcoin" ?

File: README.md [hunk 2]
```diff
@@ -38,6 +38,7 @@ place to ask about it might be in [ipfs/apps](https://github.com/ipfs/apps) or
 * [Cohort](https://github.com/zignig/cohort) - A golang app to preset a threejs interface and get all of its assets out of IPFS.
 * [dapple](https://github.com/nexusdev/dapple) - Dapple is a Solidity developer multitool designed to manage the growing complexity of interconnected smart contract systems.
 * [digx](https://www.dgx.io/) - Digix is an asset-tokenisation platform built on Ethereum and IPFS.
+* [dtube](dtube.video) - Distributed video sharing with steem.it intergrations, using ipfs for backend storage.
```
Comment (RichardLitt) [inline README.md]: Can you add the protocol to this link?

**Comment by RichardLitt** [comment]
Just did it myself. Merged. Thank you!

**Pull request closed without merging**